source/Xunit.Gherkin.Quick.UnitTests/GherkinScenarioOutlineTests.cs

```diff
@@ -307,12 +307,578 @@ public void ApplyExampleRow_Digests_Row_Values_Into_Scenario_With_DocString_In_S
+            "I declare: \"-5 <-6\"",                          // givenResult2
+            "the result for -5 and -6 should be False")]      // thenResult2
+        [InlineData(
+            "I declare: \"<a>< <b>\"",                        // givenExpression
+            "the result for <a> and <b> should be <result>",  // thenExpression
+            "I declare: \"5< 6\"",                            // givenResult1
+            "the result for 5 and 6 should be True",          // thenResult1
+            "I declare: \"-5< -6\"",                          // givenResult2
+            "the result for -5 and -6 should be False")]      // thenResult2
+        [InlineData(
+            "I declare: \"<a> <= <b>\"",                      // givenExpression
+            "the result for <a> and <b> should be <result>",  // thenExpression
+            "I declare: \"5 <= 6\"",                          // givenResult1
+            "the result for 5 and 6 should be True",          // thenResult1
+            "I declare: \"-5 <= -6\"",                        // givenResult2
+            "the result for -5 and -6 should be False")]      // thenResult2
+        [InlineData(
+            "I declare: \"<a><=<b>\"",                        // givenExpression
+            "the result for <a> and <b> should be <result>",  // thenExpression
+            "I declare: \"5<=6\"",                            // givenResult1
+            "the result for 5 and 6 should be True",          // thenResult1
+            "I declare: \"-5<=-6\"",                          // givenResult2
+            "the result for -5 and -6 should be False")]      // thenResult2
+        [InlineData(
+            "I declare: \"<a> <=<b>\"",                       // givenExpression
+            "the result for <a> and <b> should be <result>",  // thenExpression
+            "I declare: \"5 <=6\"",                           // givenResult1
+            "the result for 5 and 6 should be True",          // thenResult1
+            "I declare: \"-5 <=-6\"",                         // givenResult2
+            "the result for -5 and -6 should be False")]      // thenResult2
+        [InlineData(
+            "I declare: \"<a><= <b>\"",                       // givenExpression
+            "the result for <a> and <b> should be <result>",  // thenExpression
+            "I declare: \"5<= 6\"",                           // givenResult1
+            "the result for 5 and 6 should be True",          // thenResult1
+            "I declare: \"-5<= -6\"",                         // givenResult2
+            "the result for -5 and -6 should be False")]      // thenResult2
+        public void ApplyExampleRow_Digests_Row_Values_Into_Scenario_With_Operators_And_Spaces(
+            string givenExpression,
+            string thenExpression,
+            string givenResult1,
+            string thenResult1,
+            string givenResult2,
+            string thenResult2)
```

Comment:
Please rename these `xxxResultN` variables. It took me 15 minutes to understand what these were.
Call these `expectedxxxExpressionWithExampleRowN`. Where `xxx := Given | Then`, and `N := 1 | 2`

e.g., `expectedGivenExpressionWithExampleRow1`. Such variable names will be easier to understand since they are more intention-revealing. If you have even better names in mind, go for it.

Reply:
Done.